### PR TITLE
Add laminas support

### PR DESCRIPTION
**Pull request by javabudd**
References https://github.com/Codeception/module-zf2/issues/9

### CODE REVIEW

**Comment by javabudd** [comment]
Should this be a new repository or a major tag? IMO a new repository would make more sense because of the package name.

File: src/Codeception/Lib/Connector/Laminas.php [hunk 1]
```diff
@@ -1,143 +1,117 @@
...
         if ($queryString) {
-            parse_str($queryString, $query);
+            \parse_str($queryString, $query);
         }
 
         if ($method !== HttpRequest::METHOD_GET) {
             $post = $request->getParameters();
         }
 
+        $zendRequest->setCookies(new Parameters($request->getCookies()));
```
Comment (Naktibalda) [inline src/Codeception/Lib/Connector/Laminas.php]: Please replace everything `zend` to `laminas`.

**Review by Naktibalda — CHANGES_REQUESTED** [review]
I apologize for leaving you in the dark for so long, but I will get to this eventually.

I will clone module-zf2 to module-laminas and will merge your change to it.

**Comment by javabudd** [comment]
@Naktibalda it's no problem, I've updated the zend references to `laminas`

File: composer.json [hunk 2]
```diff
@@ -15,9 +15,12 @@
     ],
     "minimum-stability": "RC",
     "require": {
-        "php": ">=5.6.0 <8.0",
+        "php": "^7.1",
```
Comment (javabudd) [inline composer.json:18]: Should we default this to something higher?

Comment (Naktibalda) [reply composer.json:18]: All laminas dependencies support PHP 5.6, so it would be the best to support PHP 5.6 here too.

**Comment by Naktibalda** [comment]
https://packagist.org/packages/codeception/module-laminas

**Comment by Naktibalda** [comment]
Please test "module-laminas": "dev-master", I will release 1.0.0 later.

**Comment by Naktibalda** [comment]
I released module-laminas 1.0.0 in November.

codeception.com website was updated to show Laminas module instead of ZF2: https://codeception.com/docs/modules/Laminas

**Pull request closed without merging**